generated/proto/namespace/namespace.pb.go

```diff
@@ -18,28 +18,27 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-// Code generated by protoc-gen-gogo. DO NOT EDIT.
+// Code generated by protoc-gen-go.
```

Comment:
I think you need to rebase off master and re-gen this file

Reply:
just tried that, didn't see any changes. Is the DO NOT EDIT a new thing?

Reply:
You sure you've pulled changes for the `ci-scripts` submodule too? Justin changed the underlying generator to be using gogo instead of vanilla protobuf a couple of days ago.

Reply:
I'll try pulling master on the ci-scripts subdirectory